dev/dll/ResourceHelper.cpp

```diff
@@ -14,15 +14,15 @@ void EnsureLibraryLoaded();
 void SetProcessMuiLanguages();
 
 void LoadStringResource(
-    _In_ HINSTANCE module,
+    HINSTANCE module,
```

Comment:
We don't use these functions anymore (only using MRT resources) so maybe we just delete these?